src/server/nyplApiClient/index.ts

```diff
@@ -8,8 +8,7 @@ interface KMSCache {
   clientId: string
   clientSecret: string
 }
-
-const appEnvironment = process.env.APP_ENV || "production"
+const appEnvironment = process.env.APP_ENV || "development"
```

Comment:
I believe we still want the fallback to be production, is this here for local testing?

Reply:
I must have misunderstood our conversation. I thought we agreed on this change but maybe it was just temporarily for debugging.

Reply:
i returned it to production